lib/reports/establishment-conditions/index.js

```diff
@@ -0,0 +1,32 @@
+const { omit } = require('lodash');
+
+const parse = establishment => establishment.authorisations.map(a => ({
+  ...omit(establishment, 'authorisations'),
+  authorisation_type: a ? a.type : null,
+  authorisation_method: a ? a.method : null,
+  authorisation_description: a ? a.description : null
+}));
+
+module.exports = ({ db }) => {
+
+  const query = () => {
+    const q = db.asl('establishments')
+      .select(
+        'establishments.id',
+        'establishments.name',
+        'establishments.licence_number',
+        'establishments.status',
+        'establishments.conditions',
+        db.asl.raw('JSON_AGG(authorisations) as authorisations')
```

Comment:
It seems a bit weird that we're aggregating the authorisations into a single array and then mapping them back into distinct rows in `parse`. Would it be much simpler to not do the `JSON_AGG` and then have the `parse` function be a no-op?

Reply:
That's because it is weird and I'm not sure what the hell was going on there. I must've copied and pasted the file from another report and just started editing it rather than writing it from scratch.